protobuf/syft_proto/execution/v1/type_wrapper.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package syft_proto.execution.v1;
+option java_package = "org.openmined.syftproto.execution.v1";
+
+message ClassType {
+    /*
+        A type contains the serialization id and string bytes.
+    */
+    int32 id = 1;
+    bytes type = 2;
```

Comment:
If these are actually strings (and not binary data), I'd make the type `string`.